daemon/algod/api/algod.oas2.json

```
@@ -2973,15 +2973,15 @@
           "type": "integer"
         },
         "local-state-schema": {
-          "description": "[\\lsch\\] local schema",
+          "description": "\\[lsch\\] local schema",
           "$ref": "#/definitions/ApplicationStateSchema"
         },
         "global-state-schema": {
-          "description": "[\\gsch\\] global schema",
+          "description": "\\[gsch\\] global schema",
           "$ref": "#/definitions/ApplicationStateSchema"
         },
         "global-state": {
-          "description": "[\\gs\\] global schema",
+          "description": "\\[gs\\] global schema",
```

minor: is this a typo?
```suggestion
          "description": "\\[gs\\] global state",
```

---

Yes, let's fix that while we're in here. Thanks.